storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -57,6 +59,8 @@
     private int maxSchedulingAttempts;
     private int schedulingTimeoutSeconds;
     private ExecutorService backgroundScheduling;
+    private Map<String, Set<String>> evictedTopologiesMap = new HashMap<>();   // topoId : topoEvicted
+    private Map<String, Set<String>> tmpEvictedTopologiesMap = new HashMap<>();
```

Comment:
This doesn't have to be a attribute since it's only used inside schedule method

Reply:
I was thinking not to add more parameters to he private scheduleTopology function, but I think this makes sense and fixed.